libs/frontend/feature-build/src/components/build-render/ProseMirrorEditor.vue

```diff
@@ -68,7 +68,13 @@ watch(
 </script>
 
 <style lang="postcss">
-.component-content-container > p,
+/* Style when ProseMirror editor is not active */
+.component-content-container > p + p:empty::before {
+  content: '\00a0';
+  color: transparent;
```

Comment:
Hi @sampullman 

It seems that when ProseMirror editor is active, a `<br>` will be used as a new line, which utilizes the current `font-size` as height.
Since we also want to show a new line of the same height when ProseMirror editor is not active, I use `::before` to add an empty character to the empty `<p>`, which will have the same height as `<br>`.

Reply:
Sounds good!